electricitymap/contrib/config/__init__.py

```diff
@@ -1,137 +1,59 @@
+"""Global config variables with data read from the config directory."""
+
 from collections import defaultdict
 from copy import deepcopy
 from pathlib import Path
-from typing import Any, Dict, List, NewType, Tuple
+from typing import Dict, List
 
 import yaml
```

Comment:
This import is no longer used.

Reply:
Thanks, done. Btw is there a tool that you use/recommend for automatically checking for unused imports?

Reply:
Note, autoflake catches the unused imports; flake8 (which is used in current lint process) also should be able to catch it, but apparently doesn't catch it currently.